src/java.desktop/share/classes/java/awt/Component.java

```diff
@@ -4693,6 +4694,9 @@ public void setIgnoreRepaint(boolean ignoreRepaint) {
     }
 
     /**
+     * Checks whether or not paint messages received from the operating system
```

Comment:
In the change above the "Returns whether or not" text is used. It is better to use the same new text, or just use the old one.

Reply:
Done